.github/workflows/java8_integration_tests_ft.yml

```
@@ -0,0 +1,65 @@
+name: Java 8 Integration Tests
+
+on: [pull_request]
+
+jobs:
+  build:
+    name: "modules: "
+
+    strategy:
+      fail-fast: false
+      matrix:
+        modules:
+          - >-
+            alluxio.server.ft.**,!alluxio.server.ft.journal.raft.**,!alluxio.server.ft.journal.ufs.**
+          - >-
+            alluxio.server.ft.journal.raft.**,alluxio.server.ft.journal.ufs.**
+
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.pull_request.title, 'DOCFIX') &&
+        !contains(github.event.pull_request.title, 'SKIPCI')"
+
+    steps:
+      - name: checkout repo
+        uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: '10.11.0'
+
+      - name: Cache local Maven repository
+        uses: actions/cache@v2
+        with:
+          path: ~/.m2/repository
+          key: ${{ runner.os }}-maven-java8-${{ hashFiles('**/pom.xml') }}
+
+      - name: Cache local Go modules
+        uses: actions/cache@v2
+        with:
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-gomod-${{ hashFiles('**/go.mod') }}
+
+      - name: Run tests
+        id: test0
+        run: |
+          mkdir -p ~/.m2
+          ALLUXIO_DOCKER_FORK_COUNT=1 \
```

add a comment to highlight the importance of this line
```suggestion
          # set fork count to 1 because ....
          ALLUXIO_DOCKER_FORK_COUNT=1 \
```